src/protocol/attribution/aggregate_credit.rs

```diff
@@ -194,6 +196,10 @@ where
                                 let step = BitOpStep::from(check_idx);
                                 let c = c2.narrow(&step);
                                 let record_id = RecordId::from(i);
+                                /// 
+                                /// TODO: modulus convert "check" to an element in the large prime field here!
```

Comment:
@martinthomson - I'm stuck here. Need your help figuring out how to perform the conversion at this point.